### PR TITLE
Center zoom on mouse

### PR DESCRIPTION
**Pull request by Dimava**

### CODE REVIEW

**Comment by Dimava** [comment]
Will rebase this on `master` if asked after `master` becomes runnable

**Comment by Dimava** [comment]
Closes #346

**Comment by Dimava** [comment]
The feature was beta-tested
image

File: src/js/game/camera.js [hunk 1]
```diff
@@ -509,6 +510,16 @@ export class Camera extends BasicSerializableObject {
 
         this.clampZoomLevel();
         this.desiredZoom = null;
+
+        const mousePosition = this.root.app.mousePosition;
+        if (mousePosition) {
+            const worldPos = this.root.camera.screenToWorld(mousePosition);
+            let de = worldPos.sub(this.center);
```
Comment (tobspr) [inline src/js/game/camera.js:517]: Could you find a more meaningful name and make it const? :)

**Comment by Dimava** [comment]
Should I rebase it on master?

**Comment by Dimava** [comment]
Should I rebase it on master?

**Comment by tobspr** [comment]
would be awesome :)

**Comment by Dimava** [comment]
...I did something wrong it seems 
will try again

**Comment by tobspr** [comment]
Thank you!

**Pull request merged**